templates/rust/events/src/config.rs

```diff
@@ -0,0 +1,26 @@
+use actix_web::web;
+
+/// Run custom configuration as part of the application building
+/// process.
+///
+/// This function should contain all custom configuration for your function application.
+///
+/// ```rust
+/// fn config(cfg: &mut web::ServiceConfig) {
+///     let db_driver = my_db();
+///     cfg.data(db_driver.clone());
+/// }
+/// ```
+///
+/// Then you can use configured resources in your function.
+///
+/// ```rust
+/// pub async fn handle(
+///     event: Event,
+///     driver: web::Data<DbDriver>,
+/// ) -> Result<Event, actix_web::Error> {
+///     Ok(Event::default())
+/// }
+pub fn configure(_cfg: &mut web::ServiceConfig) {
+    log::info!("Configuring service");
+}
```

Comment:
Might be nice to avoid the `_` prefix and use the arg in the body. I don't think `ServiceConfig` implements `Debug` but `AppConfig` should...

```suggestion
pub fn configure(cfg: &mut web::ServiceConfig) {
    log::info!("Configuring service from {:?}", cfg.config());
}
```

Reply:
I couldn't find a good example for this, service config doesn't expose any info that could be printed (`config()` function doesn't exists). Maybe adding 'health' route, but to me that should belong to the main. This

Reply:
Hmm... I might've been looking at v4 of actix. Is there a `to_string()` or something?

Reply:
Maybe we can introduce a really simple configuration for our example function, like

https://github.com/dejanb/kn-plugin-func/commit/95294bf3fe0d53b9746d2ee78600d5ed2ae19a3f

I think that might be a good way to get people get started more easily.

If you think it's a good approach, I can follow up with something like this in a separate PR.

Reply:
works for me. the simpler, the better!